src/app/contextBroker/orionRestServices.cpp

```diff
@@ -551,6 +552,7 @@ void orionRestServicesInit
    const char*         httpsCert
 )
 {
+  //Use options service vector (optionsServiceV) only when CORS is enabled
```

Comment:
Ah ... the comment was moved here ... ?
If so ... All OK (I guess) , but the Orion style guide 'demands' `//` to be followed by a space

Reply:
Yes I moved the comment here. It seemed to make more sense to have the comment where the decision is made.

Fixed in: b87cdd8